src/urllib3/contrib/securetransport.py

```diff
@@ -165,6 +166,21 @@
     )
 
 
+_tls_version_to_st: Dict[int, int] = {
+    ssl.TLSVersion.MINIMUM_SUPPORTED: SecurityConst.kTLSProtocol1,
+    ssl.TLSVersion.TLSv1: SecurityConst.kTLSProtocol1,
+    ssl.TLSVersion.TLSv1_1: SecurityConst.kTLSProtocol11,
+    ssl.TLSVersion.TLSv1_2: SecurityConst.kTLSProtocol12,
+    ssl.TLSVersion.MAXIMUM_SUPPORTED: SecurityConst.kTLSProtocol12,
+}
+# Reverse mapping for mapping back to TLSVersion once set.
+_st_to_tls_version = {
+    v: k
+    for k, v in _tls_version_to_st.items()
+    if k not in (ssl.TLSVersion.MINIMUM_SUPPORTED, ssl.TLSVersion.MAXIMUM_SUPPORTED)
+}
```

Comment:
Do we use this somewhere?

Reply:
Nope! Was used in a previous implementation but survived the change. Great catch, thank you :)

Reply:
Removed this construction, going to merge now :)